hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/spark/sql/hudi/command/AlterHoodieTableRenameCommand.scala

```diff
@@ -46,6 +45,15 @@ class AlterHoodieTableRenameCommand(
 
       // Call AlterTableRenameCommand#run to rename table in meta.
       super.run(sparkSession)
+
+      // update table properties path in every op
+      if (hoodieCatalogTable.catalogProperties.contains("path")) {
+        val catalogTable = sparkSession.sessionState.catalog.getTableMetadata(newName)
+        val path = catalogTable.storage.locationUri.get.getPath
+        logInfo(s"alter ${oldName} name to ${newName}, update tblp 'path' to ${path}")
```

Comment:
How about `Alter table name from {oldName} to {newName}, update table property of 'path' to {path}`

Reply:
@KnightChess 
You thumbed up on this comment. Would you please update the log accordingly ?
From my side, it's also fine to just remove it -- Renaming a table should update the path property by default, it's ok no logging. But it depends on you :)